extensions/granola/package.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://www.raycast.com/schemas/extension.json",
   "name": "granola",
   "title": "Granola",
-  "version": "1.4",
+  "version": "1.5",
```

Comment:
Is it worth removing the version from the package.json since we don't really use it and it's just another place to keep up to date?